index.html

```diff
@@ -596,14 +781,35 @@ <h2>Contact</h2>
         }, 250)
       );
 
-      canvas.addEventListener("mousemove", draw);
+      let timeoutReference;
+
+      canvas.addEventListener("mousemove", (e) => {
```

Comment:
instead of directly calling draw when moving, i start an interval to draw every frame essentially